RubberduckTests/Grammar/SelectionExtensionsTests.cs

```diff
@@ -637,5 +638,58 @@ End If
             Assert.IsFalse(selection.Contains(contexts.ElementAt(1)));  // first outer if block
             Assert.IsTrue(selection.Contains(contexts.ElementAt(2)));   // second outer If block
         }
+
+        [TestMethod]
+        [TestCategory("Grammar")]
+        [TestCategory("Selection")]
+        public void Selection_Token_BlankLines_Contains()
```

Comment:
Hmm, I can see `GivenOnlyNewLines_EndColumnWorks` and `GivenOnlyNewLines_EndLineWorks` tests happening - both `EndColumn()` and `EndLine()` are under test here, no?

Reply:
Yes, for both start and end tokens as well.

Reply:
The tokens are just data - we're really testing that the extension methods produce the expected results; there should be one extension method being tested per test IMO.